components/menu/menu-item-link.js

```diff
@@ -70,14 +70,6 @@ class MenuItemLink extends MenuItemMixin(LitElement) {
 		`;
 	}
 
-	updated(changedProperties) {
-		super.updated(changedProperties);
-		if (changedProperties.has('text')) {
-			// Voiceover + iOS can't find label inside <a>
-			this.setAttribute('aria-label', this.text);
```

Comment:
Haha what a familiar change this is... made by me 2 months ago!

Reply:
Hahaha I didn't realize this was added so recently, it feels kinda nice to remove the setAttribute and just move this directly over to a property though!